CIP-0086/README.md

```diff
@@ -686,24 +671,6 @@ from a given address in an indexer
 than to keep track of all the people who control various authorization tokens,
 at a given time.
 
-### Restricting CIP-86 update transactions
```

Comment:
@nalane instead of simply striking out this section, can you leave the explanation of why the "non-minting transaction" requirement was originally dictated, but then (equally importantly) explain why that requirement was dropped?

Reply:
Hi @rphair, I have added the section as requested

Reply:
thanks @nalane (resolved with https://github.com/cardano-foundation/CIPs/pull/521/commits/c7583bc08a844249db634e8930a14de3034db42d)